dotnet/aspnetcore22/containerWithTests/Application/aspnet-core-dotnet-core.UnitTests/SampleUnitTests.cs

```diff
@@ -0,0 +1,18 @@
+using aspnet_core_dotnet_core.Pages;
+using Microsoft.AspNetCore.Mvc.Razor;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+    
+namespace aspnet_core_dotnet_core.UnitTests
+{
+    [TestClass]
+    public class SampleUnitTests
+    {
+        [TestMethod]
+        public void IndexPageTest()
+        {
+            IndexModel model = new IndexModel();
+            Assert.AreEqual("Index", actual: model.DoTest());
```

Comment:
Instead of testing a unused mehtod, you can test the OnGet method

Reply:
The OnGet() returns nothing - there is nothing to test .  
I think what is there is good enough to illustrate how to add tests when a real app is built out.